docs/reference/content/driver/tutorials/ssl.md

```diff
@@ -222,3 +222,18 @@ updates:
 
 Note that these updates are only available from Oracle via its Java SE commercial support program.  Java 7 Update 131
 is available via [OpenJDK](http://openjdk.java.net/install/).
+
+### Forcing TLS 1.2
+
+Some applications may want to force only the TLS 1.2 protocol. To do this, get an instance of [`javax.net.ssl.SSLContext`](https://docs.oracle.com/javase/8/docs/api/javax/net/ssl/SSLContext.html) that implements the TLS 1.2 protocol and set the `socketFactory` property, as in:
+
+```java
+SSLContext sslContext = SSLContext.getInstance("TLSv1.2");
```

Comment:
When I run this example, this exception is thrown:

```
Exception in thread "main" java.lang.IllegalStateException: SSLContext is not initialized
	at java.base/sun.security.ssl.SSLContextImpl.engineGetSocketFactory(SSLContextImpl.java:208)
	at java.base/javax.net.ssl.SSLContext.getSocketFactory(SSLContext.java:311)
	at JAVA3386.main(JAVA3386.java:30)

```

Were you able to get this to run?

Reply:
I did not include a call to `sslContext.init(null, null, null);` after getting the instance. However, the use of the system property is a much more elegant solution to this, and I have updated the documentation accordingly.